x-pack/plugins/stack_connectors/common/bedrock/schema.ts

```diff
@@ -35,6 +35,8 @@ export const InvokeAIActionParamsSchema = schema.object({
   temperature: schema.maybe(schema.number()),
   stopSequences: schema.maybe(schema.arrayOf(schema.string())),
   system: schema.maybe(schema.string()),
+  // abort signal from client
+  signal: schema.maybe(schema.any()),
```

Comment:
where is this being used that `signal` is validated as well? (I would expect validation to be used for requests and responses and `signal` is something we inject)

Reply:
It's likely being validated by the framework, when the action is requested to be executed.  We always validate before running ...